src/executables/util_cs.c

```diff
@@ -4649,9 +4654,75 @@ memmon (UTIL_FUNCTION_ARG * arg)
       goto error_exit;
     }
 
+  if (!print_transaction && !print_module && !print_show_all)
+    {
+      print_default = true;
+    }
+
+  /* execute phase */
+  error_code = mmon_get_server_info (server_info);
+  if (error_code != NO_ERROR)
+    {
+      goto error_exit;
+    }
+
+  // TODO: mmon_print_server_info (server_info);
+
+  if (module_index >= 0)
```

Comment:
Nit:

Using print_module flag is better than module_index.

```
if (print_module)
{
    assert (module_index >= 0 && module_index < MMON_MODULE_LAST);
}
else
{

}
```

Reply:
I agree. Done.